ps1/ps1/src/twitter/Extract.java

```diff
@@ -1,177 +0,0 @@
-            
-        } else if(at_char >= text.length()) {
-            throw new RuntimeException("character index out of range of text length.");
-            
-        } else if(at_char == 0){
-            // if it's the first character of the tweet,
-            // we add an unacceptable preceding character to avoid errors
-            char_preceding = '#';
-            char_succeeding = text.charAt(at_char+1);
-            
-        } else if (at_char == text.length()-1) {
-            // if it's the last character of the tweet,
-            // we add an unacceptable succeeding character to avoid errors
-            char_preceding = text.charAt(at_char-1);
-            char_succeeding = '#';
-
-        } else {
-            // for all other character positions,
-            // we just enter the true preceding/succeeding characters
-            char_preceding = text.charAt(at_char-1);
-            char_succeeding = text.charAt(at_char+1);
-        }
-        
-        // test if it's a username - preceding char is invalid,
-        // and succeeding char is valid
-        
-        if ((AcceptableChars.indexOf(char_preceding) == -1) &&
-            (AcceptableChars.indexOf(char_succeeding) != -1) ){
-                            
-            // get the full string of the username
-            StringBuilder username = new StringBuilder();
-            
-            // add characters to username while characters are valid
-            at_char = at_char + 1;
-            while ((at_char < text.length()) && 
-                    (AcceptableChars.indexOf(text.charAt(at_char)) != -1)){
-                username.append(text.charAt(at_char));
-                at_char = at_char + 1;
-            }
-            
-            return username.toString().toLowerCase();
-            
-        } else {
-            return "";
-        }
-        
-    }
-
-    /**
-     * Get usernames mentioned in a list of tweets.
-     * 
-     * @param tweets
-     *            list of tweets with distinct ids, not modified by this method.
-     * @return the set of usernames who are mentioned in the text of the tweets.
-     *         A username-mention is "@" followed by a Twitter username (as
-     *         defined by Tweet.getAuthor()'s spec). The username-mention cannot
-     *         be immediately preceded or followed by any character valid in a
-     *         Twitter username. For this reason, an email address like
-     *         bitdiddle@mit.edu does NOT contain a mention of the username mit.
-     *         Twitter usernames are case-insensitive, and the returned set may
-     *         include a username at most once.
-     */
-    public static Set<String> getMentionedUsers(List<Tweet> tweets) {
-        
-
-        Set<String> UserList = new HashSet<String>();
-        
-        for (int i = 0; i < tweets.size(); i++){
-                        
-            String t = tweets.get(i).getText();
-            
-            int at_char = 0;
```

Comment:
super small comment, you don't really need that placeholder no ? DRY 😄

Reply:
you mean just write `int at_char ;` ? Or drop the line altogether?